Makefile

```diff
@@ -73,6 +73,7 @@ str_match_test:
 # testing
 ##
 test: joy joy_test.py
+	./download_test_pcaps.sh test_pcaps/pcapr
```

Comment:
Will this fail the tests automatically if there isn't access to the Internet to download these files?